notebooks/spacy_eval_en.ipynb

```diff
@@ -1,5 +1,15 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Download the model before attempting to run this NB\n",
```

Comment:
oh, i see the instructions are here. Since this applies to the repo itself and not just this notebook, it should be repeated in the indicated places

Reply:
gotcha!